### PR TITLE
Do not number images saved using `save()`

### PR DESCRIPTION
**Pull request by isthisnagee**
Hi all! First time on this codebase, so please feel free to point out any issues with the code!

There's also [this todo](https://github.com/p5py/p5/blob/91d88affb5495b19923e2941d2a2e58b78d48327/p5/sketch/userspace.py#L342-L345) related to the filename, so I'm happy to discuss an approach that would solve both.

### CODE REVIEW

**Comment by johntiger1** [comment]
👍 🚀

File: p5/sketch/userspace.py [hunk 1]
```diff
@@ -311,9 +311,7 @@ def save(filename='screen.png'):
     :type filename: str
 
     """
-    # TODO: images saved using ``save()`` should *not* be numbered.
-    # --abhikpal (2018-08-14)
-    p5.sketch.screenshot(filename)
+    p5.sketch.screenshot(filename, numbered_fname=False)
```
Comment (parsoyaarihant) [inline p5/sketch/userspace.py:314]: Does this setting matches the ones in Processing? 

Maybe you can add default attribute in the `def save(filename='screen.png')` as well

Comment (isthisnagee) [reply p5/sketch/userspace.py:314]: Oh, I didn’t even consider what the setting was in Processing😅

I’ll make it consistent!

**Comment by ziyaointl** [comment]
Since this PR hasn't been updated for a while, I'll close it for now. If anyone wants to pick it up, please feel free to reopen.

**Pull request closed without merging**